charts/govuk-rails-app/values.yaml

```diff
@@ -0,0 +1,70 @@
+# Default/example values for a Rails application.
+
+service:
+  type: NodePort
+  annotations: {}
+  port: 80
+
+externalSecrets:
+  enabled: true
+  refreshInterval: 60m
+
+# enable true if Application requires public facing endpoint
```

Comment:
This is just the standard Helm ingress template from the default starter chart, so doesn't really need a comment. (The default starter chart doesn't have a comment here either.)
```suggestion
```